frontend/src/data/Dialect.ts

```diff
@@ -0,0 +1,27 @@
+export const Dialect: Dialect = {
+  dialectToName: {
+    "draft2020-12": "Draft 2020-12",
+    "draft2019-09": "Draft 2019-09",
+    draft7: "Draft 7",
+    draft6: "Draft 6",
+    draft4: "Draft 4",
+    draft3: "Draft 3",
+  },
+  dialectMapping: {
```

Comment:
These don't look right. Please double check them. Specifically, some JSON Schema dialect IDs have empty fragments (`#`) and some do not. Make sure you've copied them exactly as-is from the Python file or from the specification page.